test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
yeah, I am totally lost on why and where is it picking the ASPNET package from. I will open a bug with respect to this scenario.

I was looking at some of the sample cli applications and they started using "netcoreapp1.0" TFM. Please let me know if that is the way to go, and should I change the TFM from "netstandardapp1.5" to the new one. Thanks.